services/src/Microsoft.Azure.IIoT.Services.Common.Configuration/src/Startup.cs

```diff
@@ -124,6 +125,7 @@ public class Startup {
             }
 
             app.UseSwagger();
+            app.UseMetricServer();
```

Comment:
`UseMetricServer()` provides an overload with port number as well. Can we please use it in all places and use the same port for all components, let's say `8080` or something else ?

Reply:
Creating a task for this.